pkg/reconcilermanager/controllers/util.go

```diff
@@ -181,6 +185,15 @@ func reconcilerEnvs(clusterName, syncName string, syncGeneration int64, reconcil
 		},
 	)
 
+	if requiresNSController {
```

Comment:
Is the conditional necessary? For `RenderingEnabled` we always set the env var, including for a value of `False`.

I assume it doesn't make much functional difference since the reconciler defaults to false, but WDYT?

Reply:
Yes, I was also thinking about whether to add the condition for `requiresRendering` when implementing this.
In the flags, we explicitly set the default to false when env var doesn't exist. I feel like we don't need to set the default in two places.